### DISCUSSION TITLE
Fill dynamic dropdown

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5384] [Q&A] [answered]
I have blueprint call categories it has just name field, on the other side i created new blueprint call products and i want to generate select (dropdown list) fill with the defined categories as i see when i add new select field for product it just let me fill it with defined items. any help to make this please ?

**Comment by ryanmitchell** [comment, marked as answer]
Did you look into the entries field type? It should allow you to select items from your categories collection.

**Reply by HazemMohamed252** [reply]
Sorry how can i find this ? Can you explain more please ?

**Reply by ryanmitchell** [reply]
I assume your categories are set up as a collection? If so then in your products blueprint add a field type "Entries". It lets you select items from collections you specify.
<img width="592" alt="Screenshot 2022-03-02 at 09 32 12" src="https://user-images.githubusercontent.com/51899/156334435-56d95d2e-ae15-4736-9829-5d5e556a7659.png">
.

**Reply by HazemMohamed252** [reply]
My categories not set as a collection, it is outside the collection in separate menu, should i make it as collection ?

**Reply by ryanmitchell** [reply]
I'm not sure what you mean - can you explain your setup some more?

**Reply by HazemMohamed252** [reply]
I have created new resource using runway addon, a new list has been created in the sidebar for CP call products and created another one call orders, these lists not under the collection it is globals and assests, so as i try to understand from you , you mesn that i should put it under the collection menu to add entries ?

**Reply by ryanmitchell** [reply]
No, if you are using Runway then use the BelongsTo field type it provides to select your categories.

**Reply by HazemMohamed252** [reply]
Thanks so much. really many thanks for your help. I did it.
